build.gradle

```diff
@@ -0,0 +1,107 @@
+buildscript {
+  ext.kotlin_version = "[1.2,)"
+
+  dependencies {
+    repositories {
+      mavenCentral()
+      jcenter()
+    }
+    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    classpath "org.jetbrains.dokka:dokka-gradle-plugin:0.9.17"
```

Comment:
I think this won't be needed if you update to gradle 5.2.1 and use the more recent plugin syntax I mention below.

Reply:
correct, removed